docs/installation-configuration.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: Mist Provider Installation & Configuration
-meta_desc: Provides an overview on how to configure the Pulumi Mist Provider.
+title: Juniper Mist Provider Installation & Configuration
+meta_desc: Provides an overview on how to configure the Pulumi Juniper Mist Provider.
 layout: package
 ---
 
 ## Installation
 
-The mist provider is available as a package in all Pulumi languages:
+The junipermist provider is available as a package in all Pulumi languages:
```

Comment:
I think this should actually be in "English" (since the non-english name varies in each language) ... so, Juniper Mist seems more appropriate.

Note also that Zach also submitted a partial fix for some of this stuff below ...

Reply:
Sounds good. I can make that change. 

Zach's fix was directly to the registry; these are the source files that are used to update the registry on pulbish

Reply:
There are also some grammatical "errors" in the file (not in this PR) ... for example:

This version is supporting the following Mist Clouds:

should be

This version supports the following Mist Clouds:

so if we can give a more careful review and fix everything in one go, that would be lovely!